env.example.yml

```diff
@@ -7,7 +7,7 @@ CSV_ENABLED: true
 CUSTOM_PELIAS_URL: http://<insert your Pelias endpoint here>/v1
 GEOCODER: HERE # Options: HERE, PELIAS
 TRANSIT_GEOCODER: (optional) OTP/PELIAS
-TRANSIT_BASE_URL: (conditionally required) OTP instance when TRANSIT_GEOCODER=OTP
+TRANSIT_BASE_URL: (conditionally required) OTP instance when TRANSIT_GEOCODER=OTP (/otp/routers/{routerId})
```

Comment:
I think router id is going away in OTP, so do we need to put that example in the template?

Reply:
For now, it's helpful as a reminder of what otp-ui appends and doesn't append